src/client/api/event.c

```diff
@@ -182,6 +182,13 @@ daos_eq_free(struct d_hlink *hlink)
 	D_FREE(eq);
 }
 
+crt_context_t
+daos_get_crt_ctx()
```

Comment:
oh, because I do not want to use task in dc_mgmt_profile, since we will never need task interface for dc_mgmt_profile(). So I decided to create daos_send_rpc_sync() here to send rpc without task.  I think we will use that for those management api and other test api.